gwlfe/enums.py

```diff
@@ -8,62 +8,98 @@ class YesOrNo(object):
     NO = '<No>'
     YES = '<Yes>'
 
-    @staticmethod
-    def parse(value):
+    @classmethod
+    def parse(cls, value):
         if value in ('0', 'N'):
-            return YesOrNo.NO
+            return cls.NO
         elif value in ('1', 'Y'):
-            return YesOrNo.YES
+            return cls.YES
+        raise ValueError('Unexpected value: ' + str(value))
+
+    @classmethod
+    def intval(cls, value):
+        if value == cls.NO:
+            return 0
+        elif value == cls.YES:
+            return 1
+        raise ValueError('Unexpected value: ' + str(value))
+
+    @classmethod
+    def gmsval(cls, value):
+        if value == cls.NO:
+            return 'N'
+        elif value == cls.YES:
+            return 'Y'
         raise ValueError('Unexpected value: ' + str(value))
 
 
 class ETflag(object):
     HAMON_METHOD = '<Hamon method>'
     BLAINY_CRIDDLE_METHOD = '<Blainy-Criddle method>'
 
-    @staticmethod
-    def parse(value):
+    @classmethod
+    def parse(cls, value):
         value = int(value)
         if value == 0:
-            return ETflag.HAMON_METHOD
+            return cls.HAMON_METHOD
         elif value == 1:
-            return ETflag.BLAINY_CRIDDLE_METHOD
+            return cls.BLAINY_CRIDDLE_METHOD
+        raise ValueError('Unexpected value: ' + str(value))
+
+    @classmethod
+    def gmsval(cls, value):
```

Comment:
Adding `intval` for other flags was kind of a hack. I'd like to avoid it here since the value of this flag is symbolic and not used in any calculation.